docs/releasenotes/release-1.127.md

```diff
@@ -1,35 +1,49 @@
-# v1.126.0
+* Special shout-outs to @acpana, @anhdle-sso, @cheftako, @ericpang777, @gemmahou, @haiyanmeng, @jasonvigil, @jingyih, @justinsb, @maqiuyujoyce, @nb-goog, @xiaoweim, @yuwenma, @ziyue-101 for their contributions to this release.
 
-** This version is not yet released; this document is gathering release notes for the future release **
+## Announcement 
 
-* ...
 
-* Special shout-outs to ... for their contributions to this release.
-TODO: list contributors with `git log v1.126.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+## New Beta Resources (Direct Reconciler):
 
-## Announcement 
+* [`BigQueryAnalyticsHubListing`](https://cloud.google.com/config-connector/docs/reference/resource-docs/bigqueryanalyticshub/bigqueryanalyticshublisting)
```

Comment:
Thanks for adding this 🙏🏼 I was going to open a PR to add it too 😆 

FWIW, 

/lgtm